Cargo.toml

```diff
@@ -33,7 +33,7 @@ ignore = { version = "0.4.22" }
 regex = { version = "1.10.4" }
 serde = { version = "1.0.200", features = ["derive"] }
 serde_yaml = "0.9.33"
-tree-sitter = { version = "0.24.5", package = "tree-sitter-facade-sg" }
+tree-sitter = { version = "0.25.0-codemod.2", package = "tree-sitter-facade-sg-codemod" }
```

Comment:
Just a copy of the facade package with the new methods `node_kind_is_supertype` and `supertypes` exposed.